content/soarian/r4/individuals/patient.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Patient | R4 API
+---
+
+# Patient
+
+* TOC
+{:toc}
+
+## Overview
+
+The Patient resource provides general demographic information about a person receiving health care services from a specific organization. Common demographic fields include patient ID, patient name, gender, date of birth, address, phone, and primary language. Additional concepts returned as extensions and not part of the base resource include race, ethnicity, and birth sex. Soarian Clinicals<sup>®</sup> is a patient-centric application; thus, many of its other resources include the patient ID in their queries. A person receiving care from multiple organizations may have data available in multiple patient resources in multiple Fast Healthcare Interoperability Resources<sup>®</sup> (FHIR)<sup>®</sup> servers.
```

Comment:
@slvsridhar, As in my previous comment, please change the fourth sentence so it reads as follows. (Replace ";thus," with ",so")
Soarian Clinicals<sup>®</sup> is a patient-centric application, so many of its other resources include the patient ID in their queries.

Reply:
Done